docs/kuneiform/sql-as-understood-by-kwil/functions.mdx

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_position: 2
+sidebar_label: "Supported Functions"
+id: supported-functions
+title: Supported SQL Functions
+description: Supported SQL functions in Kuneiform.
+slug: /kuneiform/functions
+---
+
```

Comment:
Add a note that functions can only be called in procedures?

Reply:
They can be called in select statements. Will add some docs to actions